src/main/java/com/example/demo/discountComponent/api/dtos/GetDiscountsDTO.java

```diff
@@ -0,0 +1,20 @@
+package com.example.demo.discountComponent.api.dtos;
+
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+
+import java.util.List;
+
+@Getter
+@Builder
+public class GetDiscountsDTO {
+    @NotNull
+    private final int totalItems;
+    @NotNull
+    private final int totalPages;
+    @NotNull
+    private final int currentPage;
```

Comment:
int tipo reiksmiu nebutina zymet kaip not null, nes jie negali but null, nebent naudotum Integer (not a class = can't be null)